.gitignore

```diff
@@ -12,3 +12,4 @@ _javadocs/
 code2yaml/
 api/.inter/
 .openpublishing.buildcore.ps1
+.idea
```

Comment:
Is is necessary?

Reply:
It's necessary for Intellij users. Let's keep it.